src/main/scala/org/broadinstitute/dsde/firecloud/core/ExportEntitiesByType.scala

```diff
@@ -93,14 +93,14 @@ class ExportEntitiesByTypeActor(requestContext: RequestContext) extends Actor wi
       Future(RequestComplete(StatusCodes.BadRequest)) pipeTo context.parent
   }
 
-  private def getZipBytes(filename: String, membershipData: String, rootType: String, rootData: String): Array[Byte] = {
+  private def getZipBytes(entityType: String, membershipData: String, entityData: String): Array[Byte] = {
     val bos = new ByteArrayOutputStream()
     val zos = new ZipOutputStream(bos)
-    zos.putNextEntry(new ZipEntry(filename))
+    zos.putNextEntry(new ZipEntry(entityType + "_membership.txt"))
     zos.write(membershipData.getBytes)
     zos.closeEntry()
-    zos.putNextEntry(new ZipEntry(rootType + ".txt"))
-    zos.write(rootData.getBytes)
+    zos.putNextEntry(new ZipEntry(entityType + "_entity.txt"))
```

Comment:
I was erring on the side of clarity since this is an entity file and not an update file.